theme/tutorial.less

```diff
@@ -216,6 +216,13 @@ i.icon.avatar-image {
     }
 }
 
+.tutorialhint .ui.segment .blocklyPreview {
+    width: 100%;
+    @media only screen and (min-height: 400px) {
```

Comment:
do we have a breakpoint variable defined in our LESS for this?

Reply:
no, i copied this from the previous modal height breakpoint (css above this line), but i can move it into the less? not sure how it was previously determined